node/src/Pos/Launcher/Scenario.hs

```diff
@@ -107,7 +103,6 @@ runNode' NodeResources {..} workers' plugins' = ActionSpec $ \vI sendActions ->
     tipHeader <- DB.getTipHeader @ssc
     logInfo $ sformat ("Current tip header: "%build) tipHeader
 
-    initSemaphore
```

Comment:
So this is a refactoring that does not change behaviour because `runNode'` is called within `bracketNodeResources` which calls `allocateNodeContext` where we now initialise the `slTip :: MVar` and so we do not need to set it here.